.ci/benchmarks.groovy

```diff
@@ -8,10 +8,8 @@ pipeline {
     BASE_DIR = "src/github.com/elastic/${env.REPO}"
     AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'
     EC_KEY_SECRET = 'secret/observability-team/ci/elastic-cloud/observability-pro'
-    BENCHMARK_ES_SECRET = 'secret/apm-team/ci/benchmark-cloud'
-    KIBANA_SECRET = 'secret/k8s/elastic-apps/oauth-proxy/prod/domains/observability-benchmarks.elastic.dev'
+    BENCHMARK_ES_SECRET = 'secret/apm-team/ci/benchmark-cloud'    
```

Comment:
I'd say to use `secret/observability-team/ci/benchmark-cloud` instead, 

`secret/apm-team/ci/benchmark-cloud` is deprecated and all the references in our docs point to the above one. 


```
status                  DEPRECATED, use secret/observability-team/ci/benchmark-cloud instead
```

I'm still concerned with the existing user/pass, is it possible to create a new user with only the right permissions to query the `kibana_dashboard_url` and `png` generation, in favour of reducing the chances of any security disclosure?